zk_toolbox/crates/common/src/withdraw.rs

```diff
@@ -0,0 +1,305 @@
+use async_trait::async_trait;
+// use serde_json::Value;
+use ethers::{
+    providers::{JsonRpcClient, Provider, ProviderError},
+    types::Bytes,
+    types::{Address, TransactionReceipt, TxHash, H160, H256, U64},
+};
+use serde::{Deserialize, Serialize};
+use serde_json::json;
+use zksync_system_constants::L1_MESSENGER_ADDRESS;
+use zksync_types::ethabi;
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct Proof {
```

Comment:
same here, it seems to me that it can be imported from the `core` directory

Reply:
where?